Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM mozillamarketplace/centos-mysql-mkt:0.2
 
+# Set the locale. This is mainly so that tests can write non-ascii files to
+# disk.
+ENV LANG en_US.UTF-8
```

Comment:
nah, going with international "English" like the rest of the world

/me ducks deeper